Shell/AST.cpp

```diff
@@ -367,9 +367,15 @@ RefPtr<Value> BarewordLiteral::run(RefPtr<Shell>)
 void BarewordLiteral::highlight_in_editor(Line::Editor& editor, Shell& shell, HighlightMetadata metadata)
 {
     if (metadata.is_first_in_list) {
-        editor.stylize({ m_position.start_offset, m_position.end_offset }, { Line::Style::Bold });
+        if (shell.is_runnable(m_text))
+            editor.stylize({ m_position.start_offset, m_position.end_offset }, { Line::Style::Bold });
+        else {
+            editor.stylize({ m_position.start_offset, m_position.end_offset }, { Line::Style::Foreground(Line::Style::XtermColor::Red) });
```

Comment:
Let's keep it bold either way, it's nice to have an indicator that doesn't depend on the text itself.

Also, I'm not too sure about the color here, red is also used to highlight syntax errors...but no other good alternative comes to mind, feel free to disregard this bit.

Reply:
Bold red seems a bit in your face IMO. Are you sure? I'll push it if you want though.

![serenity-terminal](https://user-images.githubusercontent.com/15224242/89166186-fd645380-d5bc-11ea-93ab-1b2b791a2f96.png)

Reply:
Well...the non-bold variant looks better, I guess.
let's leave it non-bold then.